src/app/extensions/quoting/facades/quoting.facade.ts

```diff
@@ -58,6 +58,13 @@ export class QuotingFacade {
     );
   }
```

Comment:
above: please move `this.store.dispatch(loadQuoting());` into a separate facade method `loadQuoting()` and reuse it in the effect above and in the new `quoting-basket-line-items.component`, that way it is not that implicitly used.